content/docs/users/_index.md

```diff
@@ -9,7 +9,17 @@ type : "docs"
 Users can begin the journey with PySAL by exploring the courses,
 workshops, tutorials, and presentations:
 
-### Courses, Workshops, and Tutorials
+### Books
+-   [Rey, S., Arribas-Bel, D., & Wolf, L. J. (2023). Geographic data science with python. Chapman and Hall/CRC.](https://geographicdata.science/book/intro.html)
+-   [Anselin, L., & Rey, S. J. (2014). Modern spatial econometrics in practice: A guide to GeoDa, GeoDaSpace and PySAL](https://www.google.com/books/edition/Modern_Spatial_Econometrics_in_Practice/MBLwrQEACAAJ?hl=en)
+
+### Courses
+-   [Spatial Data Science for Social Geography](https://martinfleischmann.net/sds/): a course developed and taught in person at Charles University in Prague in 2023 by Martin Fleischmann
```

Comment:
```suggestion
-   [Spatial Data Science for Social Geography](https://martinfleischmann.net/sds/): a course developed and taught in person at Charles University in Prague since 2023 by Martin Fleischmann
```

Reply:
Thanks! Changed.